infra/gcp-broad/main.tf

```diff
@@ -175,6 +175,17 @@ resource "google_container_cluster" "vdc" {
   }
 
   timeouts {}
+
+  addons_config {
+    network_policy_config {
+      disabled = false
+    }
+  }
+
+  network_policy {
+    enabled = true
+    provider = "CALICO"
```

Comment:
Why are you using Calico for this? And what is the policy that will be applied?

Reply:
That is the setting supported by Google cloud. AFAIK I'm not applying a policy, just enabling the concept of network policies, as requested by CIS